okhttp/src/main/java/okhttp3/CipherSuite.java

```diff
@@ -29,356 +32,421 @@
  * <p>See also <a href="https://android.googlesource.com/platform/external/conscrypt/+/master/src/main/java/org/conscrypt/NativeCrypto.java">NativeCrypto.java</a>
  * from conscrypt, which lists the cipher suites supported by Android.
  */
-public enum CipherSuite {
+public final class CipherSuite {
+  // for interning - must be at top of file
```

Comment:
Nit: we follow this: http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html